src/app/services/applicationSettings/application-settings.service.ts

```diff
@@ -8,11 +8,15 @@ import { BackendService } from '../backend/backend.service';
 @Injectable({
   providedIn: 'root'
 })
+
 export class ApplicationSettingsService {
+  public edited: boolean = false;
```

Comment:
What does edited means here. This doesn't clarify the right meaning of the variable.

We need to specify the right name, also we don't need this variable.

Rather you can directly have variables like selectedTeamId and selectedSprintId.

And change it in application service, whenever needed.

Reply:
"edited" variable is used to check if the sprint was edited after the application was launched or not.

Reply:
You can check it by comparing old and new values.

If they are not equal then they are edited.
else not edited.